### ISSUE TITLE
[Bug]: @supports in CSS crashes Chromium

### ISSUE THREAD

**Issue by guillaumebrunerie** [microsoft/playwright#33172]
### Version

1.48.1

### Steps to reproduce

1. Clone https://github.com/guillaumebrunerie/playwright-crash-supports.
2. Run `npm i`.
3. Run `npx http-server` in a different terminal (make sure you get port 8080 as it is hardcoded in the test).
4. Run `npx playwright test`.

### Expected behavior

No crash.

### Actual behavior

Chromium crashes, while Firefox and Webkit work fine.

```
Running 3 tests using 3 workers
  1) [chromium] › example.spec.js:4:1 › has title ──────────────────────────────────────────────────

    Error: page.goto: Page crashed
    Call log:
      - navigating to "http://localhost:8080/", waiting until "load"


      3 |
      4 | test('has title', async ({ page }) => {
    > 5 |   await page.goto('http://localhost:8080');
        |              ^
      6 |
      7 |   // Expect a title "to contain" a substring.
      8 |   await expect(page).toHaveTitle(/Test/);

        at /Users/guillaumeb/playwright-crash-supports/tests/example.spec.js:5:14

  1 failed
    [chromium] › example.spec.js:4:1 › has title ───────────────────────────────────────────────────
  2 passed (1.8s)
```

### Additional context

What makes it crash is the following CSS line in `index.html`:

```css
@supports (color: oklch(from #123456 max(L, 0.75) C H)) {}
```

Note that when testing manually both with Chrome stable (129) and Chrome beta (130) the page works fine. It worked fine in the past as well.

### Environment

```
  System:
    OS: macOS 15.0
    CPU: (8) arm64 Apple M2
    Memory: 70.50 MB / 16.00 GB
  Binaries:
    Node: 21.6.1 - ~/.nvm/versions/node/v21.6.1/bin/node
    Yarn: 1.22.22 - ~/.nvm/versions/node/v21.6.1/bin/yarn
    npm: 10.9.0 - ~/.nvm/versions/node/v21.6.1/bin/npm
    pnpm: 9.12.2 - ~/.nvm/versions/node/v21.6.1/bin/pnpm
    bun: 1.0.25 - ~/.bun/bin/bun
  IDEs:
    VSCode: 1.86.0 - /Applications/Visual Studio Code.app/Contents/Resources/app/bin/code
  Languages:
    Bash: 3.2.57 - /bin/bash
  npmPackages:
    @playwright/test: ^1.48.1 => 1.48.1
```

**Comment by guillaumebrunerie** [comment]
After testing a bit more, it doesn’t have to do with `@supports` but seems to be an oddly specific bug with min/max in relative colors:

```css
body {
  background-color: rgb(from #123456 max(R, 200) G B); /* Crashes */
  background-color: rgb(from #123456 calc(max(R, 200)) G B); /* Crashes */
  background-color: rgb(from #123456 calc(R + 200) G B); /* Works */
}
```

**Comment by dgozman** [comment]
Filed an upstream issue: https://issues.chromium.org/u/1/issues/374244474.

**Comment by guillaumebrunerie** [comment]
Looks like the upstream issue was fixed last week, when can we expect it to be fixed in Playwright? Is it for Playwright 1.49?

**Comment by dgozman** [comment]
@guillaumebrunerie The fix will probably be available in v1.50, because it takes a few weeks for the Chromium change to end up in a release.

**Comment by dgozman** [comment]
Confirmed this is fixed on `main` and will be available in v1.50.
